ui/apps/platform/src/configureApolloClient.js

```diff
@@ -55,6 +55,9 @@ export default function configureApolloClient() {
                             // the resolver will cause duplicate requests and lost cache data.
                             merge: (existing, incoming) => merge({}, existing, incoming),
                         },
+                        name: {
```

Comment:
If `image->name` is requested multiple times on the same page, but with different sub-fields, extra network requests will be sent and cache data will be lost. This ensures that image name fields are merged in the client side cache.

Reply:
good catch! could you add this as a comment so the logic doesn't get lost?